### DISCUSSION TITLE
Filter engine for backlinks, graph, page list...

### DISCUSSION THREAD

**Discussion by Limezy** [athensresearch/athens#1028] [Feature Requests]
I'm sure it's in the roadmap somewhere but I just put it here for discussion.
Obsidian have been doing some pretty impressive stuff recently in their graph filtering capabilities

**Comment by Ejipuh** [comment]
What would the filter engine do?

**Comment by datokrat** [comment]
In designing a query language, it's probably advisable not to forget about #845, so that there is only one query language.
